bgpd/bgp_evpn_vty.c

```diff
@@ -4114,6 +4251,62 @@ DEFUN(show_bgp_l2vpn_evpn_route_vni_all,
 	return CMD_SUCCESS;
 }
 
+/*
+ * Display per-VNI EVPN routing table in detail - for all VNIs.
+ */
+DEFUN(show_bgp_l2vpn_evpn_route_vni_all_detail,
```

Comment:
I guess evpn_show_routes_vni_all command is already present. Why can't we extend that command with "detail" as optional key word ? Any specific reason to create entirely new command ?

Reply:
I would tend to agree here -- it's better to extend a command than to create a new one to show a more detailed view of already existing information.

Reply:
Thanks for the comment. I do see the sense in it. Will make the necessary changes to extend evpn_show_routes_vni_all. A separate command was created because there were things like "header" getting displayed as part of evpn_show_routes_vni_all.

Reply:
Uploaded a new patch with the required changes. Could you please take a look?